patterns/state/ships/src/main/java/com/scottlogic/dp/state/ships/FishingShip.java

```diff
@@ -3,6 +3,8 @@
 public class FishingShip {
     private ShipState state = new DriftingState();
 
+    Engine engine = new Engine();
+
     void setState(ShipState state) {
```

Comment:
Could this can be private/protected now? :)

Reply:
Ok :)